packages/@okta/vuepress-site/docs/guides/implement-grant-type/index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Implement authorization by grant type
```

Comment:
nit: this should be title case: http://www.conissaunce.com/pages/infoq-title-case.html

Reply:
I'm following the Okta dev docs style guide to use sentence case for headings/titles: https://github.com/okta/okta-developer-docs/wiki/Style-Guide#capitalization